src/query/owner/errors/domain-exception.ts

```diff
@@ -0,0 +1,5 @@
+export class DomainException extends Error {
```

Comment:
Is er verschil tussen de `sensor/domain-exception.ts` en deze class? Zou dat dezelfde kunnen zijn?